packages/server/lib/browsers/firefox.ts

```diff
@@ -676,6 +681,21 @@ export async function open (browser: Browser, url: string, options: BrowserLaunc
       return browserReturnStatus || driverReturnStatus
     }
 
+    // In some cases, the webdriver session will NOT return the moz:debuggerAddress capability even though
+    // we set it to true in the capabilities. This is out of our control, so when this happens, we fail the browser
+    // and gracefully terminate the related processes and attempt to relaunch the browser in the hopes we get a
+    // CDP address.
+    if (!webdriverClient.capabilities['moz:debuggerAddress']) {
+      debugVerbose(`firefox failed to spawn with CDP connection. Failing current instance and retrying`)
```

Comment:
Why verbose?

Reply:
I think I had the whole error in here originally. Makes sense to just make it a regular debug statement now